docs/getting-started-guides/azure.md

```diff
@@ -35,10 +35,10 @@ Getting started on Microsoft Azure
 
 **Table of Contents**
 
-    - [Prerequisites](#prerequisites)
-    - [Setup](#setup)
-    - [Getting started with your cluster](#getting-started-with-your-cluster)
-    - [Tearing down the cluster](#tearing-down-the-cluster)
+- [Prerequisites](#prerequisites)
```

Comment:
These TOCs could be converted to auto-generated ones by inserting:
```
**Table of Contents**
 <!-- BEGIN MUNGE: GENERATED_TOC -->
...
<!-- END MUNGE: GENERATED_TOC -->
```
around them